lib/util/api-json.js

```diff
@@ -63,7 +63,11 @@ function api(token, endpoint, method, requestBody, callback) {
                 if (responseBody.trim().length === 0) {
                     responseBody = null;
                 } else {
-                    responseBody = JSON.parse(responseBody);
+                    try {
+                        responseBody = JSON.parse(responseBody);
+                    } catch (e) {
+                        return callback(new APIError(e, responseBody, responseCode));
```

Comment:
FWIW, the way I would handle this is to move the JSON parsing lower down, after the "200-299" check, so that we have a single `callback()` invocation, with either the successful response body or the `APIError`.